File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/entities/cs/createupdateentitymetadata3.cs
// <snippetcreateupdateentitymetadata3>


 CreateAttributeRequest createBalanceAttributeRequest = new CreateAttributeRequest
 {
  EntityName = _customEntityName,
  Attribute = new MoneyAttributeMetadata
  {
   SchemaName = "new_balance",
   RequiredLevel = new AttributeRequiredLevelManagedProperty(AttributeRequiredLevel.None),
   PrecisionSource = 2,
   DisplayName = new Label("Balance", 1033),
   Description = new Label("Account Balance at the last known date", 1033),

  }
 };

 _serviceProxy.Execute(createBalanceAttributeRequest);

// </snippetcreateupdateentitymetadata3>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/entitiesvb/vb/createupdateentitymetadata3.vb
' <snippetcreateupdateentitymetadata3>


 Dim createBalanceAttributeRequest As CreateAttributeRequest = New CreateAttributeRequest With {
  .EntityName = _customEntityName,
  .Attribute = New MoneyAttributeMetadata With {
   .SchemaName = "new_balance",
   .RequiredLevel = New AttributeRequiredLevelManagedProperty(AttributeRequiredLevel.None),
   .PrecisionSource = 2,
   .DisplayName = New Label("Balance", 1033),
   .Description = New Label("Account Balance at the last known date", 1033)
  }
 }

 _serviceProxy.Execute(createBalanceAttributeRequest)

' </snippetcreateupdateentitymetadata3>